consensus/docker-compose-files/scripts/infiniteloop.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Hi @tuand27613, sorry I didn't know infiniteloop.sh has been moved.

Anyway, I added back that script in that position because the cli container was failing saying the script was not found. How should we set that path?